ektelo/hdmm/more_templates.py

```diff
@@ -0,0 +1,107 @@
+from ektelo.hdmm import templates
```

Comment:
I think that would be fine.  The only reason they are separate is because the more_templates is experimental code while the stuff we've used in templates has been used quite a bit.  Also there may be an issue with merging: more_templates requires autograd.numpy while templates just uses ordinary numpy.  In principle I'm not opposed to it though